bin/clover/src/commands/generateSiSpecs.ts

```diff
@@ -59,6 +68,7 @@ export async function generateSiSpecs(
   }
 
   // EXECUTE PIPELINE STEPS
+  specs = ignoreSpecsWithoutHandlers(specs);
```

Comment:
Did we checkin the file with this in it or am I blind?

Reply:
We didn't :) I just added it.